src/transformers/models/bert/modeling_bert.py

```diff
@@ -120,7 +120,14 @@ def load_tf_weights_in_bert(model, config, tf_checkpoint_path):
         # adam_v and adam_m are variables used in AdamWeightDecayOptimizer to calculated m and v
         # which are not required for using pretrained model
         if any(
-            n in ["adam_v", "adam_m", "AdamWeightDecayOptimizer", "AdamWeightDecayOptimizer_1", "global_step"]
+            n
+            in [
+                "adam_v",
+                "adam_m",
+                "AdamWeightDecayOptimizer",
+                "AdamWeightDecayOptimizer_1",
+                "global_step",
```

Comment:
Hey, I still see a lot of formatting changes in this PR that will probably propagate through the codebase when we `make fix-copies`! Do you have something like a save hook for `PyCharm` that always runs `black`?

Reply:
This looks much better, thank you!